DEVELOPMENT.md

```diff
@@ -11,6 +11,8 @@ $ source tf/bin/activate
 
 TensorBoard builds are done with [Bazel](https://bazel.build), so you may need to [install Bazel](https://docs.bazel.build/versions/master/install.html). The Bazel build will automatically "vulcanize" all the HTML files and generate a "binary" launcher script. When HTML is vulcanized, it means all the script tags and HTML imports are inlined into one big HTML file. Then the Bazel build puts that index.html file inside a static assets zip. The python HTTP server then reads static assets from that zip while serving.
 
+For any changes to the frontend, you’ll need to install [Yarn][yarn] to lint your code (`yarn lint`, `yarn fix-lint`). You’ll also need Yarn to add or remove any NPM dependencies.
```

Comment:
Maybe move this down to just after the next paragraph + example?  The previous paragraph is about building w/ bazel and the next paragraph is the concrete example of that, so it's nice to keep them together.  Linting and adding/removing NPM deps seems a little less critical so it could go further down.

Reply:
Done.